system-int-test/test/cmr/system_int_test/search/autocomplete_collection_facet_search_test.clj

```diff
@@ -0,0 +1,81 @@
+(ns cmr.system-int-test.search.autocomplete-collection-facet-search-test
+  "Integration tests for autocomplete collection search facets"
+  (:require
+    [clojure.test :refer :all]
+    [cheshire.core :as json]
+    [clojurewerkz.elastisch.rest :as esr]
+    [cmr.common.util :as util :refer [are3]]
+    [cmr.system-int-test.data2.core :as d]
+    [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+    [cmr.system-int-test.utils.ingest-util :as ingest]
+    [cmr.system-int-test.utils.index-util :as index]
+    [cmr.system-int-test.utils.search-util :as search]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(deftest autocomplete-suggest-test
+  (let [conn (esr/connect "http://localhost:9210")
```

Comment:
OK for now, but we want this test to be updated with real collection with the instruments of the value.

Reply:
I'll need to have the harvest feature in place first, but I agree. I'd prefer to not POST to ES directly in any case.